cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -386,6 +386,17 @@ message MinorLines {
     repeated LineDescriptor ranges = 1;
 }
 
+message SeismicTraceGroupExtent {
+    oneof type {
```

Comment:
So this `oneof` is for future-proofing? Might as well, but I think we can put the `group_header` in the `SeismicTraceGroupExtent` message?

Reply:
Yeah just for future proofing, thinking we might want a Def variant at some point specifying the minor trace header field. Can move the group header to here